### PR TITLE
Add brazilian portuguese language

### PR DESCRIPTION
**Pull request by mjsf12**
in alphabetic order

### CODE REVIEW

File: compatibility.html [hunk 1]
```diff
@@ -222,14 +231,15 @@
...
+                            <span ng-if="lang == 'br'"><span class="flag-icon flag-icon-br"></span> BR</span>
                             <span ng-if="lang == 'en'"><span class="flag-icon flag-icon-gb"></span> EN</span>
                             <span ng-if="lang == 'fr'"><span class="flag-icon flag-icon-fr"></span> FR</span>
                             <span ng-if="lang == 'gr'"><span class="flag-icon flag-icon-gr"></span> GR</span>
                             <span ng-if="lang == 'it'"><span class="flag-icon flag-icon-it"></span> IT</span>
                             <span ng-if="lang == 'nl'"><span class="flag-icon flag-icon-nl"></span> NL</span>
                             <span ng-if="lang == 'pl'"><span class="flag-icon flag-icon-pl"></span> PL</span>
                             <span ng-if="lang == 'vn'"><span class="flag-icon flag-icon-vn"></span> VN</span>
-                        </a>
+                            </a>
```
Comment (IllusionMan1212) [inline compatibility.html]: Indentation.
sorry for nitpicking

File: compatibility.html [hunk 2]
```diff
@@ -247,7 +257,9 @@ <h5 class="modal-title" id="exampleModalLabel">{{texts[19]}}</h5>
                     </button>
                 </div>
                 <div class="modal-body">
-                   <a href="#" data-dismiss="modal" ng-click="changeLang(1)"><span class="flag-icon flag-icon-sa"></span> العربية</a>
+                    <a href="#" data-dismiss="modal" ng-click="changeLang(1)"><span class="flag-icon flag-icon-sa"></span> العربية</a>
+                    <br>
+                    <a href="#" data-dismiss="modal" ng-click="changeLang(9)"><span class="flag-icon flag-icon-br"></span> Brasileiro</a>
```
Comment (IllusionMan1212) [inline compatibility.html]: change the number 9 to 2 and all the others too in numerical order

File: index.html [hunk 3]
```diff
@@ -208,6 +218,8 @@ <h5 class="modal-title" id="exampleModalLabel">{{texts[33]}}</h5>
                 <div class="modal-body">
                     <a href="#" data-dismiss="modal" ng-click="changeLang(1)"><span class="flag-icon flag-icon-sa"></span> العربية</a>
                     <br>
+                    <a href="#" data-dismiss="modal" ng-click="changeLang(9)"><span class="flag-icon flag-icon-br"></span> Brasileiro</a>
```
Comment (IllusionMan1212) [inline index.html]: same thing for the number

**Comment by mjsf12** [comment]
you are like my programming teacher about this, no problem ,anything else?

**Pull request merged**